ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -5454,7 +5454,7 @@ components:
 
     PartyContactDetailWithPattern:
       type: object
-      title: Party Contact Detail
+      title: Party Contact Detail (with phone pattern)
```

Comment:
Does the 'phone pattern' make sense if you can also supply an email address? At least, I try to match this change with the description below.